lib/interface_quda.cpp

```diff
@@ -8998,8 +8921,10 @@ void calcMG_loop_wOneD_TSM_wExact(void **gaugeToPlaquette,
 	t1 = MPI_Wtime();	
 	K_vector->downloadFromCuda(sol,flag_eo);
 	K_vector->download();
+
```

Comment:
The following is a misleading comment. It's not any exact part that is projected out of the solution. The solution itself (the spinor "x") gets projected with (1-P), as the original comment stated.